### DISCUSSION TITLE
Welcome to bluesound Discussions!

### DISCUSSION THREAD

**Discussion by aunefyren** [aunefyren/bluesound_alt#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by carryonrewardless** [comment]
Hi
Thanks for bluesound_alt.  I am loving it with my NAD MDC BluOS 2i module and being able to select it as a player in HA Music Assistant.
Do you know of a way to enable this module to wake on LAN?  I power the NAD amp up through HA with a smart plug but it comes up in standby so still need to grab the remote control to get things started.
Cheers

**Reply by aunefyren** [reply]
Hey! Thank you for the fine kind words. I suppose that would be possible, as long as the API can do that in some way. Would that for example be a power toggle in HA which wakes the player?

**Comment by carryonrewardless** [comment]
Yeah I just use a wifi connected smart plug to toggle the power between on/off from the HA dashboard.

**Comment by carryonrewardless** [comment]
Hi. An update. I am still getting used to the HA Music Assistant and just found that the BluOS 2i module powers up from standby when a music track is sent to it. So nothing more to do on this issue.
Thanks again for your work.
